### ISSUE TITLE
Removing helm chart should remove all tensorboards and notebook

### ISSUE THREAD

**Issue by maver1ck** [polyaxon/polyaxon#156]
### Describe the bug

After removing helm chart (--purge) there are still active pods from polyaxon:
```
NAME                                                              READY     STATUS        RESTARTS   AGE
plx-notebook-654ecad3d8cc46de9337ebb5fe04c7b9-fb6cdcb95-4btkr     1/1       Running       0          2d
plx-tensorboard-05158fc55d0a46e1b2ed20b6eb3e951a-5d5cbb947ljfwn   1/1       Running       0          2d
```

### To Reproduce
- create polyaxon deployment with helm
- use tensorboard or jupyter
- delete deployment

### Expected behavior
All pods will be deleted

### Environment
Polyaxon 0.1.5

**Comment by maver1ck** [comment]
I think the problem is with hooks
```
Events:
  Type     Reason                 Age                 From                                               Message
  ----     ------                 ----                ----                                               -------
  Normal   Scheduled              54m                 default-scheduler                                  Successfully assigned brawny-unicorn-clean-experiments-vqnsf to gke-concourse-default-pool-2cd18e9b-rmtg
  Normal   SuccessfulMountVolume  54m                 kubelet, gke-concourse-default-pool-2cd18e9b-rmtg  MountVolume.SetUp succeeded for volume "default-token-vm248"
  Normal   SandboxChanged         54m (x2 over 54m)   kubelet, gke-concourse-default-pool-2cd18e9b-rmtg  Pod sandbox changed, it will be killed and re-created.
  Warning  Failed                 52m (x11 over 54m)  kubelet, gke-concourse-default-pool-2cd18e9b-rmtg  Error: configmaps "brawny-unicorn-polyaxon-config" not found
  Normal   Pulled                 4m (x228 over 54m)  kubelet, gke-concourse-default-pool-2cd18e9b-rmtg  Container image "polyaxon/polyaxon-manage:0.1.6" already present on machine
```

**Comment by mouradmourafiq** [comment]
Yes that's something that I also added to the [delete section in the docs](https://docs.polyaxon.com/reference_polyaxon_helm/#uninstalling-the-chart)

**Comment by maver1ck** [comment]
Maybe `.spec.activeDeadlineSeconds` in job definition can be a solution ?

**Comment by mouradmourafiq** [comment]
Thanks for the suggestion, I will try it and report back.

**Comment by mouradmourafiq** [comment]
This was solved in the latest version 0.1.7. Please feel free to reopen.
